File: Kody-master/klasa 2/aiter_zadania/lpierwsza/lpierwsza.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    int n;
    cout << "Podaj n: ";
    cin << n;
    for (int i = 2; i * i <= n; i++)
    {
        if (n % i == 0)
        {
            cout << "Złożona";
            return 0;
        }
    }
    cout << "Pierwsza";
    
    return 0;
}

File: Kody-master/klasa 2/aiter_zadania/lpierwsza/lpierwsza.py
#! /usr/bin/env python
# -*- coding: utf-8 -*-

def main(args):
    n = int(input("Podaj liczbę całkowitą: "))
    i = 2
    while i * i <= n:
        if n % i == 0:
            print("Złożona")
            return 0
        i += 1
    print("Pierwsza")
     
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
